### PR TITLE
Added k-skip-n-gram word tokenizer

### PR DESCRIPTION
**Pull request by absolutic**
Hello @andrewdalpino, I've created new branch skip-gram-0.3.0 inheriting 0.3.0 as a parent. 
If everything is ok I will close my first PR.
Could you please update this PR with correct CLA signature allowing me to pass CLA assistant?

### CODE REVIEW

**Comment by absolutic** [comment]
_**I have read the CLA Document and I hereby sign the CLA**_

**Comment by absolutic** [comment]
recheckcla

**Comment by simplechris** [comment]
I think the CLA assistant is getting confused because the authorship of the [commit](https://github.com/RubixML/RubixML/pull/127/commits/3820edcff3cb60c2ddfb50400b02fd96a14f0db2) is attached to the [`oksana-yudenko`](https://github.com/oksana-yudenko) github account, but the cla is getting signed from the [`absolutic`](https://github.com/absolutic) github account.

File: benchmarks/Other/Tokenizers/KSkipNGramBench.php
Comment (andrewdalpino) [inline benchmarks/Other/Tokenizers/KSkipNGramBench.php]: This should be `namespace Rubix\ML\Benchmarks\Others\Tokenizers;` I believe

File: benchmarks/Other/Tokenizers/KSkipNGramBench.php
Comment (andrewdalpino) [inline benchmarks/Other/Tokenizers/KSkipNGramBench.php:1]: Nice benchmark, thanks for including

File: docs/other/tokenizers/k-skip-n-gram.md
Comment (andrewdalpino) [inline docs/other/tokenizers/k-skip-n-gram.md:4]: Very nice description

**Review by andrewdalpino — DISMISSED** [review]
Excellent PR Oksana! Other than the namespacing of the benchmark, this is good to go!

File: benchmarks/Other/Tokenizers/KSkipNGramBench.php
Comment (absolutic) [reply benchmarks/Other/Tokenizers/KSkipNGramBench.php]: Yes, thank you, I've just copy pasted other benchmark file from the same folder, fixed.
But it gave me the idea to check all other files from the same directory.

**Comment by absolutic** [comment]
> I think the CLA assistant is getting confused because the authorship of the [commit](https://github.com/RubixML/RubixML/pull/127/commits/3820edcff3cb60c2ddfb50400b02fd96a14f0db2) is attached to the [`oksana-yudenko`](https://github.com/oksana-yudenko) github account, but the cla is getting signed from the [`absolutic`](https://github.com/absolutic) github account.

Yes, I also thought in this direction.
Seems like I created second account using the same email forgetting about the first, and now github for some reason mixes them even though I am using only absolutic for all operations.

**Comment by andrewdalpino** [comment]
Is there a way we can fix it @absolutic?

**Comment by absolutic** [comment]
> Is there a way we can fix it @absolutic?

If you mean commit authoring issue I will at least try to check this hypothesis (that different commit and PR authors affect CLA bot).
Benchmark namespacing is fixed.

**Comment by andrewdalpino** [comment]
Yes I meant the issue with CLA bot, we should be good to go after that!

**Comment by absolutic** [comment]
> Yes I meant the issue with CLA bot, we should be good to go after that!

I've amended commit author here https://github.com/RubixML/RubixML/pull/128 as a part of experiment (from oksana.yudenko to absolutic) but it did not help, CLA build failed.

**Comment by andrewdalpino** [comment]
Ok no worries @absolutic I'll find a way to add you as a signatory manually

As soon as I figure that out, we'll get these merged

**Comment by andrewdalpino** [comment]
@absolutic I wasn't able to handle it *manually* ... can you fix the fact that the name on your commit is different from the name you are commenting from?

If you decide to go this route, please also pull in the latest changes since we've made changes to CLA bot since

**Comment by absolutic** [comment]
> @absolutic I wasn't able to handle it _manually_ ... can you fix the fact that the name on your commit is different from the name you are commenting from?
> 
> If you decide to go this route, please also pull in the latest changes since we've made changes to CLA bot since

@andrewdalpino Yes, will do this (amend committer name + pull latest changes + force push) on Saturday, thank you!

**Comment by andrewdalpino** [comment]
Sound good @absolutic thank you

**Comment by absolutic** [comment]
> Sound good @absolutic thank you

@andrewdalpino I did it, but this status change 

>  dismissed their stale review

cleared my hacktoberfest results :(
because only approved and tagged as "hacktoberfest-accepted" PRs are counted towards the contest and every PR is undergoing a trial period of 2 weeks after approval.

**Review by andrewdalpino — APPROVED** [review]
> cleared my hacktoberfest results :(

Oh no! Is it ok now? I re-approved the PR

**Pull request merged**
